plugins/migratecmd/migratecmd_test.go

```diff
@@ -779,3 +779,150 @@ func TestAutomigrateCollectionNoChanges(t *testing.T) {
 		}
 	}
 }
+
+func TestAutomigrateRuleDowngrade(t *testing.T) {
```

Comment:
To avoid maintaining another test, could you update the existing `TestAutomigrateCollectionUpdate`?

Reply:
done